pkg/watch/watcher_darwin.go

```
@@ -93,7 +93,7 @@ func (d *fseventNotify) Start() error {
 
 	numberOfWatches.Add(int64(len(d.stream.Paths)))
 
-	d.stream.Start()
+	d.stream.Start() //nolint:errcheck // FIXME(thaJeztah): should this return an error?
```

Left a TODO here as I wasn't sure if we could return the error